src/components/Stake/SideBar.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, textStyle, useTheme } from '@aragon/ui'
+import CardAccordion from './CardAccordion'
+import coin from './assets/coin.svg'
+import wallet from './assets/wallet.svg'
+
+const Sidebar = React.memo(function Sidebar() {
```

Comment:
Probably don't need to memoize just yet, if we feel we need it later for performance we can always come back to it!